mtp_api/apps/notification/management/commands/send_notification_emails.py

```diff
@@ -16,7 +16,7 @@
 
 class Command(BaseCommand):
     def handle(self, **options):
-        frequency = EMAIL_FREQUENCY.DAILY
+        frequency = EmailFrequency.daily
```

Comment:
Given this is pretty much a constant (AFAICT) you could move it out and call it `DAILY` (or something like that) as it may read a little bit better, e.g.

```Python
period_start, period_end = get_notification_period(DAILY)
# [...]
preferences = EmailNotificationPreferences.objects.filter(frequency=DAILY).exclude(last_sent_at=today)
```

it's fine tho, I know we're not rewriting/tidying up everything as part of this :P

Reply:
at some point in the past, it was supposed to be configurable. `EmailFrequency` is entirely redundant